parse_config/utility.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef UTILITY_HPP
+# define UTILITY_HPP
+
+# include <iostream>
+# include <string>
+
+void	exitWithErrmsg(std::string errmsg);
```

Comment:
```suggestion
class Utility {
  public :
    void	exitWithErrmsg(std::string errmsg);
}
```
과 같이 클래스로 관리하는거는 어떨까요?

Reply:
임시로 둔거긴 한데, 객체로 만들어질 필요가 없는 함수라 생각해서요. 차라리 비슷한 부류의 함수를
namespace Utility {} 이런데에 몰아 넣어서 쓰는건 어떨까요?

Reply:
이슈파두고 방법은 생각해보면 좋을거같습니다!